### DISCUSSION TITLE
19970EB4-D0E0-4A41-8622-AE71F370D916

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#204] [General]
# 19970EB4-D0E0-4A41-8622-AE71F370D916

Kepler-85, aka KOI-2038, is a G-type main-sequence star located about 2590.6 lightyears from the solar system. A system of four planets consisting of two sub-Neptunes and two super-Earths are known to orbit this star.

https://celestia.mobi/addon?item=19970EB4-D0E0-4A41-8622-AE71F370D916

**Comment by Spacespacespacey** [comment]
All I can see are cueballs!

**Reply by DaveBowman2001** [reply]
Turn off clouds in the `Settings` to see surface features of some planets
